src/interactions/panZoomInteraction.ts

```diff
@@ -59,6 +59,49 @@ namespace Plottable.Interactions {
       }
     }
 
+    /**
+     * Pans the chart by a specified amount
+     *
+     * @param {Plottable.Point} [translateAmount] The amount by which to translate the x and y scales.
+     */
+    public pan(translateAmount: Plottable.Point) {
```

Comment:
I think this is going to conflict with the changes I made to enable pan/zoom on category scales. So, can we wait until that other is merged? I can pull this down and update the PR once it's merged.